### DISCUSSION TITLE
Windows user folder, xdm system files are displayed.

### DISCUSSION THREAD

**Discussion by jm2k7** [subhra74/xdm#559] [General]
My understanding is that those XDM system files and folders should be hidden or in a location other than the user's folder view, it should be in a location away from the user's view like %appdata% or %programdata% or hidden. I understand that in linux .folder .file means hidden from the system, but in Windows the .files or .folder rule does not apply, so it is advisable to correct this cosmetic detail of the file.
![Captura de pantalla 2021-05-15 125722](https://user-images.githubusercontent.com/80860392/118372501-ead16e80-b57f-11eb-919b-a82beabf51af.jpg)

**Comment by subhra74** [comment]
It will be fixed in next release
